### DISCUSSION TITLE
Can multi-resolution hash encoding be used to fit multiple SDFs?

### DISCUSSION THREAD

**Discussion by narnia-ai-mason** [NVlabs/instant-ngp#1550] [Q&A]
Hi. I just wonder, can I use multi-resolution hash encoding to fit multiple SDFs with auxiliary latent code as input? To do so, could you suggest the hyperparameters, such as L, T, F, and size of MLP?

**Comment by nsortur** [comment]
Hello, has an answer been found for this, or are there any works that do this?
